core-bundle/src/Resources/contao/classes/Versions.php

```diff
@@ -465,7 +465,7 @@ public function compare($blnReturnBuffer=false)
 				// Find the changed fields and highlight the changes
 				foreach ($to as $k=>$v)
 				{
-					if ($from[$k] != $to[$k])
+					if (($from[$k] ?? null) != $to[$k])
```

Comment:
Wouldn’t this fail anywhere from here until line 592 where `$from[$k]` is accessed?

How about:

```suggestion
				foreach (array_keys(array_merge($to, $from)) as $k)
				{
					$to[$k] = $to[$k] ?? null;
					$from[$k] = $from[$k] ?? null;

					if ($from[$k] != $to[$k])
```

Reply:
👍 my original test case did not have data in the new field so this did not occur initially.

Reply:
Meh, it did not let me apply your suggestion unfortunately 😁